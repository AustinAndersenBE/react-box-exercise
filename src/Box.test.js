import React from 'react';
import { render } from '@testing-library/react';
import renderer from 'react-test-renderer';
import Box from './Box';

// Smoke Test
it('renders without crashing', () => {
  render(<Box id="1" handleRemove={() => {}} />);
});

// Snapshot Test
it('renders correctly', () => {
  const tree = renderer
    .create(<Box id="1" handleRemove={() => {}} />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});