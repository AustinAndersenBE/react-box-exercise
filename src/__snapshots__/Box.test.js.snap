// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <div
    style={
      {
        "backgroundColor": "blue",
        "height": "5em",
        "width": "5em",
      }
    }
  />
  <button
    onClick={[Function]}
  >
    Remove The Box!
  </button>
</div>
`;
